redbot/cogs/audio/core/commands/llset.py

```diff
@@ -59,174 +80,651 @@ async def command_llsetup_java(self, ctx: commands.Context, *, java_path: str =
+                description=_(
+                    "Managed Lavalink node will now connect using the secured {secured_protocol} protocol.\n\n"
+                    "Run `{p}{cmd}` for it to take effect."
+                ).format(
+                    p=ctx.prefix,
+                    cmd=self.command_audioset_restart.qualified_name,
+                    secured_protocol=inline("wss://"),
                 ),
             )
+        else:
+            await self.send_embed_msg(
+                ctx,
+                title=_("Setting Changed"),
+                description=_(
+                    "Managed Lavalink node will no longer connect using the secured "
+                    "{secured_protocol} protocol and wil use {unsecured_protocol} instead .\n\n"
+                    "Run `{p}{cmd}` for it to take effect."
+                ).format(p=ctx.prefix, cmd=self.command_audioset_restart.qualified_name),
+                unsecured_protocol=inline("ws://"),
+                secured_protocol=inline("wss://"),
+            )
 
     @command_llsetup.command(name="info", aliases=["settings"])
     async def command_llsetup_info(self, ctx: commands.Context):
         """Display Lavalink connection settings."""
         configs = await self.config.all()
-        host = configs["host"]
-        password = configs["password"]
-        rest_port = configs["rest_port"]
-        ws_port = configs["ws_port"]
-        msg = "----" + _("Connection Settings") + "----        \n"
-        msg += _("Host:             [{host}]\n").format(host=host)
-        msg += _("WS Port:          [{port}]\n").format(port=ws_port)
-        if ws_port != rest_port and rest_port != 2333:
-            msg += _("Rest Port:        [{port}]\n").format(port=rest_port)
-        msg += _("Password:         [{password}]\n").format(password=password)
+
+        if configs["use_external_lavalink"]:
+            msg = "----" + _("Connection Settings") + "----        \n"
+            msg += _("Host:             [{host}]\n").format(host=configs["host"])
+            msg += _("Port:             [{port}]\n").format(port=configs["ws_port"])
+            msg += _("Password:         [{password}]\n").format(password=configs["password"])
+            msg += _("Secured:          [{state}]\n").format(state=configs["secured_ws"])
+
+        else:
+            msg = "----" + _("Lavalink Node Settings") + "----        \n"
+            msg += _("Host:             [{host}]\n").format(
+                host=configs["yaml"]["server"]["address"]
+            )
+            msg += _("Port:             [{port}]\n").format(port=configs["yaml"]["server"]["port"])
+            msg += _("Password:         [{password}]\n").format(
+                password=configs["yaml"]["lavalink"]["server"]["password"]
+            )
+            msg += _("Xms:              [{xms}]\n").format(xms=configs["java"]["Xms"])
+            msg += _("Xmx:              [{xmx}]\n").format(xmx=configs["java"]["Xmx"])
```

Comment:
Perhaps it would make more sense to refer to these the same as in the llset commands - initial and max heapsize?